core/src/main/scala/chisel3/Module.scala

```diff
@@ -195,7 +195,7 @@ package internal {
         _component = _proto._component
         None
       }
-      // This module doesn't acutally exist in the FIRRTL so no initialization to do
+      // This module doesn't actually exist in the FIRRTL so no initialization to do
```

Comment:
```suggestion
      // This module doesn't ackshually exist in the FIRRTL so no initialization to do
```

Reply:
Important and hard-hitting feedback. I will take this one to heart for future work.